content/en/docs/concepts/services-networking/gateway.md

```diff
@@ -0,0 +1,207 @@
+provisioning and advanced traffic routing.
+
+<!-- body -->
+
+## Design principles
+
+The following principles shaped the design and architecture of the Gateway API:
+
+* __Role-oriented:__ Gateway API kinds are modeled after organizational roles that are
+  responsible for managing Kubernetes service networking:
+  * __Infrastructure Provider:__ Manages infrastructure that allows multiple isolated clusters
+    to serve multiple tenants, e.g. a cloud provider.
+  * __Cluster Operator:__ Manages clusters and is typically concerned with policies, network
+    access, application permissions, etc.
+  * __Application Developer:__ Manages an application running in a cluster and is typically
+    concerned with application-level configuration and [Service](/docs/concepts/services-networking/service/)
+    composition.
+* __Portable:__ Gateway API specifications are defined as [custom resources](docs/concepts/extend-kubernetes/api-extension/custom-resources)
+  and are supported by many [implementations](https://gateway-api.sigs.k8s.io/implementations/).
+* __Expressive:__ Gateway API kinds support functionality for common traffic routing use cases
+  such as header-based matching, traffic weighting, and others that were only possible in
+  [Ingress](/docs/concepts/services-networking/ingress/) by using custom annotations.
+* __Extensible:__ Gateway allows for custom resources to be linked at various layers of the API.
+  This makes granular customization possible at the appropriate places within the API structure.
+
+## Resource model
+
+The Gateway API has three stable API kinds:
+
+* __GatewayClass:__ Defines a set of gateways with common configuration and managed by a controller
+  that implements the class.
+
+* __Gateway:__ Defines an instance of traffic handling infrastructure, such as cloud load balancer.
+
+* Route kinds: The Gateway API specifies API kinds, HTTPRoute for example, that define protocol-specific
+  rules for mapping traffic from a Gateway listener to a representation of backend network endpoints.
+  These endpoints are often represented as a {{<glossary_tooltip text="Service" term_id="service">}}.
+
+The Gateway API is organized into different API kinds that have interdependent relationships to support
+the role-oriented nature of organizations. A Gateway object is associated with exactly one GatewayClass;
+the GatewayClass describes the gateway controller responsible for managing Gateways of this class.
+Route kinds such as HTTPRoute, are then associated to Gateways. A Gateway can filter the routes that
+may be attached to its `listeners`, forming a bidirection trust model with routes.
+
+The following figure illustrates the relationships of the three stable Gateway API kinds:
+
+{{< figure src="/docs/images/gateway-kind-relationships.svg" alt="A figure illustrating the relationships of the three stable Gateway API kinds" class="diagram-medium" >}}
```

Comment:
I stumbled a little over the diagram, two of the box labels are _kinds_, and the other two are presumably names of `HTTPRoutes`. I suggest changing both labels to `HTTPRoute` or add names as sort of sub title.

Reply:
I have updated the diagram based on your feedback.